### ISSUE TITLE
Performance Issue processor with FFMPEG 

### ISSUE THREAD

**Issue by openanim** [opentoonz/opentoonz#2178]
### Issue Summary

This is a problem I had for a long time. This always happens after rendering opentoonz, using ffmepg, gif for example.

The processor does not want to stop, the actions become slower.

I am forced to close opentoonz and especially ffmepg since the task manager to get things back to normal.

### System Information 
* **OpenToonz Version**:  1.2.1
* **Operating System**: Windows 10
* **CPU**:  Intel core i7 4558U CPU
* **Memory**: 8 go
* **Graphics Card**: HD graphics 5100
* **Graphics Tablet**:  cintiq companion 2

**Comment by RodneyBaker** [comment]
I'm curious what you have your FFMPEG timeout set to.

**Comment by openanim** [comment]
Indeed, it seems to be the problem, it is 1000. I had increased because of the error message that I received repeatedly.
